src/renderer/coremods/installer/AddonEmbed.tsx

```diff
@@ -0,0 +1,221 @@
+        className={svgContentLeft}
+      />
+      <path
+        d="M6.9293 10.8889C5.24647 10.8889 3.9596 9.60202 3.9596 7.91919C3.9596 6.23636 5.24647 4.94949 6.9293 4.94949C8.61212 4.94949 9.89899 6.23636 9.89899 7.91919C9.89899 9.60202 8.61212 10.8889 6.9293 10.8889ZM6.9293 20.7879C5.24647 20.7879 3.9596 19.501 3.9596 17.8182C3.9596 16.1354 5.24647 14.8485 6.9293 14.8485C8.61212 14.8485 9.89899 16.1354 9.89899 17.8182C9.89899 19.501 8.61212 20.7879 6.9293 20.7879ZM6.9293 30.6869C5.24647 30.6869 3.9596 29.4 3.9596 27.7172C3.9596 26.0343 5.24647 24.7475 6.9293 24.7475C8.61212 24.7475 9.89899 26.0343 9.89899 27.7172C9.89899 29.4 8.61212 30.6869 6.9293 30.6869Z"
+        className={svgDots}
+      />
+      <path
+        fillRule="evenodd"
+        clipRule="evenodd"
+        d="M41.4328 33L46 37.7129L43.7836 40L37 32.9999L39.2164 30.7128L39.2165 30.7129L43.7836 26L46 28.2871L41.4328 33Z"
+        className={svgTag}
+      />
+      <path
+        fillRule="evenodd"
+        clipRule="evenodd"
+        d="M54.7821 35.2871L54.7841 35.2892L57 33.0021L50.2159 26L48 28.2871L52.5662 33L48 37.7129L50.2159 40L54.7821 35.2871Z"
+        className={svgTag}
+      />
+    </svg>
+  );
+}
+
+export default function AddonEmbed({
+  addon,
+}: {
+  addon: InstallLinkProps;
+}): React.ReactElement | null {
+  if (addon.source !== "store") return null;
+
+  const [data, setData] = React.useState<CheckResultSuccess | null>(null);
+  React.useEffect(() => {
+    (async () => {
+      const info = await getInfo(addon.identifier, addon.source, addon.id);
+      if (!info) return;
+      setData(info);
+    })();
+  }, []);
+  const [onCooldown, setOnCooldown] = React.useState(false);
+  const [cooldownTimeout, setCooldownTimeout] = React.useState<NodeJS.Timeout | null>(null);
+  const [isInstalling, setIsInstalling] = React.useState(false);
+
+  if (!data) return null;
+  const { manifest } = data;
+  if (manifest.type === "replugged") return null;
+
+  const classMod = getByProps<ClassMod>("titleRegion");
+  if (!classMod) {
+    logger.error("Failed to find classMod");
+    return null;
+  }
+
+  const {
+    wrapper,
+    content,
+    title,
+    titleRegion,
+    button,
+    buttonSize,
+    icon,
+    buildInfo,
+    buildDetails,
+    subHead,
+    copyLink,
+    copyLinkIcon,
+    copied,
+  } = classMod;
+
+  const authors = authorList([manifest.author].flat().map((x) => x.name));
+  const url = getSourceLink(manifest)!; // URL should always exist since it's from store
+  const isInstalled = checkIsInstalled(data);
+  const isDisabled = isInstalled || isInstalling;
+
+  const copyUrl = (): void => {
+    window.DiscordNative.clipboard.copy(url);
+    setOnCooldown(true);
+    if (cooldownTimeout) clearTimeout(cooldownTimeout);
+    setCooldownTimeout(
+      setTimeout(() => {
+        setOnCooldown(false);
+      }, 2000),
+    );
+  };
+
+  return (
```

Comment:
Space is not optimized well, compared to the build override embed.
![image](https://github.com/replugged-org/replugged/assets/38290480/778eea07-ae40-4070-8660-6d68752209d3)

Reply:
This is somewhat still an issue, can we stretch the install button all the way to the end (if there's not enough space) just like the apply button in the screenshot above?